roles/rsyslog/tasks/inputs/basics/main.yml

```diff
@@ -6,3 +6,16 @@
     __rsyslog_rules: "{{ __rsyslog_basics_rules }}"
   include_tasks:
     file: deploy.yml
+
+- name: Create basics input configuration files in /etc/rsyslog.d
+  vars:
+    __basics_input: "{{ lookup('template', 'input_basics.j2') }}"
+    __rsyslog_packages: []
+    __rsyslog_rules: [ { "name": "input-basics-{{ basics_item.name }}", "type": "input", "weight": "11",
+                         "sections": [ { "options": "{{ __basics_input }}" } ] } ]
```

Comment:
do you need to define `__basics_input` here?  Can you do
```yaml

  vars:
    __rsyslog_packages: []
    __rsyslog_rules: [ { "name": "input-basics-{{ basics_item.name }}", "type": "input", "weight": "11",
                         "sections": [ { "options": "{{ lookup('template', 'input_basics.j2') }}" } ] } ]
```
?

Reply:
Nope, not any more. :)  Thanks!